docs/Instrumentation.md

```diff
@@ -30,19 +30,13 @@ For example, a model, mymodel.onnx, is compiled with `Debug/bin/onnx-mlir  --ins
 Its runtime output is listed below:
 
 ```
-#  0) after  op= Transpo Time elapsed: 0.000766 accumulated: 0.000766
-156608
-#  1) after  op= Constan Time elapsed: 0.005398 accumulated: 0.006164
-156608
-#  2) after  op= Constan Time elapsed: 0.004225 accumulated: 0.010389
-156608
-#  3) after  op=    Conv Time elapsed: 0.360213 accumulated: 0.370602
-156608
-#  4) after  op= Softplu Time elapsed: 0.190591 accumulated: 0.561193
-156608
-#  5) after  op=    Tanh Time elapsed: 0.115314 accumulated: 0.676507
-156608
-#  6) after  op=     Mul Time elapsed: 0.022779 accumulated: 0.699286
+#  0) after  op= Transpo Time elapsed: 0.000766 accumulated: 0.000766 VMem: 156608
+#  1) after  op= Constan Time elapsed: 0.005398 accumulated: 0.006164 VMem: 156608
+#  2) after  op= Constan Time elapsed: 0.004225 accumulated: 0.010389 VMem: 156608
+#  3) after  op=    Conv Time elapsed: 0.360213 accumulated: 0.370602 VMem: 156608
+#  4) after  op= Softplu Time elapsed: 0.190591 accumulated: 0.561193 VMem: 156608
+#  5) after  op=    Tanh Time elapsed: 0.115314 accumulated: 0.676507 VMem: 156608
+#  6) after  op=     Mul Time elapsed: 0.022779 accumulated: 0.699286 VMem: 156608
```

Comment:
That is a big improvement. I have an additional request: if by any chances we want to analyze the results, its nice to have an easy format to analyze. Here is what I have used in similar circumstances

```
@instr, op id, 0, after op, Transpo, time elapsed, 0.000766, accumulated, 0.000766, VMem, 156608
```
 namely, an easy and unique pattern to scan for `@inst` and then comma separated fields that make it easy to analyze one or more numerical fields. Still easy to read because of the text, easy to process because of the commas.